gap/PolygonalComplexes/drawing.gd

```diff
@@ -561,3 +561,418 @@ DrawSurfaceToTikz(doubleSixGon,"DoubleSixGon_edgeDraw",pr);;
 #TODO modify boundaries of page (to center picture for printing)
 #TODO for triangles, the length information should overwrite the angle information
 #TODO for >triangles can we define a notion of "this was probably meant" by minimum distance or so?
+
+
+#! @BeginChunk DrawFacegraphToTikz_Tutorial
+#! The face graph of a simplicial surface is a graph whereby the vertices of
+#! the graph are given by the faces and the edges of the graph are given by
+#! the edges of the given simplicial surface. We can compute a straight-line
+#! embedding of the face graph by assigning 2D-coordinates to the faces.
+#! Note, <K>DrawFacegraphToTikz</K> (<Ref Subsect="DrawFacegraphToTikz"/>) 
+#! only computes face graphs of spherical vertex faithful surfaces.
+#!  
+#! For example the face graph of an octahedron can be drawn like this:
+#! @BeginLog
+oct := Octahedron();;
+DrawFacegraphToTikz( oct, "facephgraph_oct_example" );;
+#! @EndLog
+#!
+#! This will write a file <E>facegraph_oct_example.tex</E> that contains the
+#! face graph of the octahedron, written in <K>TikZ</K>:
+#! <Alt Only="TikZ">
+#!     \input{Image_facegraph_oct_Rescaled.tex}
```

Comment:
You could use \input{_TIKZ_facephgraph_oct_example.tex} (also for the other images). Then not all the images have to be in doc because they are generated automatically.